onyo/lib/commands.py

```diff
@@ -675,7 +675,7 @@ def onyo_mv(inventory: Inventory,
 @raise_on_inventory_state
 def onyo_new(inventory: Inventory,
              directory: Path | None = None,
-             template: str | None = None,
+             template: Path | str | None = None,
```

Comment:
Both `Path` and `str`?

From my read, it'll always come in as `Path` or `None` now.

Reply:
From the CLI layer, yes. But if one is using the python interface directly, `str` for a template name remains valid and is tested for. See no reason to change that (yet).